tests/mod.rs

```diff
@@ -67,6 +67,33 @@ async fn trace_with_tokio_attribute() {
 }
 
 
+#[cfg(feature = "trace")]
+#[test_env_log::test]
+fn test_tracing_error() {
```

Comment:
I got confused to why some tests are copied into a "local" module.
I added only this one to test the `SpanTrace` functionality

Reply:
I think they are exercising different code paths, hence the duplication.